### PR TITLE
Content Helper: Add edit post icon and make linking behavior consistent across features

### LINKED ISSUE #1345: Makes linking behavior consistent across all features
As it stands, the Dashboard Widget’s post title link will forward to the WP Editor, while in the Content Helper sidebar (Related Top Posts) the same link forwards to the [Parse.ly](http://parse.ly/) dash. This is confusing because we have 2 links that seem to be the same thing visually, but behave differently.

In all the related areas, we should have:
1. The post title link that forwards to the [Parse.ly](http://parse.ly/) dash, which I think would be what most people expect/want when they click that link. For free VIP customers, this can also work as reminder that there’s this awesome [Parse.ly](http://parse.ly/) dash that they should purchase.
2. The existing “open” icon, which works identically everywhere currently (opens the published post).
3. A new “editor” icon (could be a pencil for example), that would open the WP Editor with the stats. Useful if someone wants to edit the post or is a free VIP user and wants to see the stats.

### PR DESCRIPTION
**Pull request by mehmoodak**
## Description
Makes linking behavior consistent across features.

Closes: #1345

## How has this been tested?
- Changes existing tests

## Screenshots (if appropriate)

1. image

2. image

### CODE REVIEW

File: src/blocks/content-helper/icons/open-link-icon.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@
  */
 import { SVG, Path } from '@wordpress/components';
 
-export const ViewsIcon = () => (
+export const OpenLinkIcon = () => (
```
Comment (mehmoodak) [inline src/blocks/content-helper/icons/open-link-icon.tsx:6]: - Changed filename from `published-link-icon` to `open-link-icon` which is more generic.
- Changed const here to match the filename

File: src/blocks/content-helper/related-top-posts/component-list-item.tsx [hunk 2]
```diff
@@ -21,9 +23,17 @@ function RelatedTopPostListItem( { post }: RelatedTopPostListItemProps ): JSX.El
 				<a className="parsely-top-post-stats-link" href={ post.dashUrl } target="_blank" rel="noreferrer" title={ __( 'View in Parse.ly (opens new tab)', 'wp-parsely' ) }>
 					{ post.title }
 				</a>
-				<a className="parsely-top-post-link" href={ post.url } target="_blank" rel="noreferrer" title={ __( 'View Published Post (opens new tab)', 'wp-parsely' ) }>
```
Comment (mehmoodak) [inline src/blocks/content-helper/related-top-posts/component-list-item.tsx:24]: Changes `View Published Post` to `View Post` which is more generic.

Just thinking that if users changes the Post from Published to draft then this text should be right.

**Review by mehmoodak — COMMENTED** [review]
Left comments for easier review.

**Review by acicovic — APPROVED** [review]
Great work!

Code and functionality look good to me, but let's wait to see if @garrettvercoe has any comments regarding design.

**Comment by acicovic** [comment]
I wasn't aware that @garrettvercoe has approved this, so feel free to merge this.

Thanks!

**Pull request merged**